score-server/src/test/java/bio/overture/score/server/service/download/ObjectDownloadServiceTest.java

```diff
@@ -82,15 +84,24 @@ public void setUp() {
     namingService.setBucketKeySize(3);
     service.setBucketNamingService(namingService);
 
-    // ReflectionTestUtils.setField(service, "dataBucketName", dataBucketName);
-    // ReflectionTestUtils.setField(service, "stateBucketName", stateBucketName);
     ReflectionTestUtils.setField(service, "dataDir", dataDir);
-    // ReflectionTestUtils.setField(service, "bucketPoolSize", 5);
-    // ReflectionTestUtils.setField(service, "bucketKeySize", 2);
     ReflectionTestUtils.setField(service, "expiration", 7);
-
     ReflectionTestUtils.setField(service, "urlGenerator", new S3URLGenerator());
     ReflectionTestUtils.setField(service, "partCalculator", new SimplePartCalculator(20000));
+
+    mockService = mock(MetadataService.class);
```

Comment:
refactor/cleanup: put all the metadataService related setup into its own method and then call it in this `setUp` method.